### DISCUSSION TITLE
appproject vs applicationset?

### DISCUSSION THREAD

**Discussion by Almenon** [argoproj/argo-cd#12501] [Q&A] [answered]
I'm new to ArgoCD. I saw that there's a applicationset that generates applications, so in some sense it's a logical grouping of applications. But the docs for appproject also say it's a logical grouping of applications, so I'm confused as to the difference between the two and when you would use them.

https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#projects
https://argo-cd.readthedocs.io/en/stable/user-guide/application-set/

**Comment by ahmedfourti** [comment, marked as answer]
The applicationset is as you said to generate applications with multiple parameters.
However, AppProject, is something else, you can think about it as "Namespace" if it is easier for you.
So basically, your applications will be part of a project, you can have one or multiple applications in one project.
If you don't specify any project, it will be in the default one.

**Reply by Almenon** [reply]
Thanks!
